src/main/java/Task.java

```diff
@@ -0,0 +1,57 @@
+import java.util.*;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Task {
+    private static List taskList = new LinkedList();
+    public static void addList(String task){
+        taskList.add(task);
+    }
+    public static void deleteList(String task){
+        taskList.remove(task);
+    }
+    public static void printList() {
+        Stream stream = taskList.stream();
+        stream.forEach(System.out::println);
+    }
+    public static void printSalute(){
+        System.out.println(
+                        "    Please select one of the following (key in number): \n" +
+                        "    1. Add a new task, \n" +
+                        "    2. Delete an old task, \n" +
+                        "    3. Show my tasks, or\n"+
+                        "    4.bye \n" +
+                        "    to end this conversation");
+    }
+    public static void exe(int instruction){
+        Scanner scan = new Scanner(System.in);
+        switch (instruction) {
+        case 1: //add task
+            System.out.println("please enter the task that you want to add(key in number): ");
+            String task1 = scan.nextLine();
+            taskList.add(task1);
+            System.out.println(task1 +" is added to your task list:)");
+            printSalute();
```

Comment:
printSalute() is called from the three, and the method's output is also present in the default case. printSalute() could be moved after the switch statement, and then the default case could only print the first line of code.